hw1/02-palindrome.html

```diff
@@ -6,20 +6,33 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
+  <body class="bg-primary">
     <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
-        </div>
-        <div>
-          <p>Is this a palindrome?</p>
-          <div></div>
+      <section class="d-flex justify-content-center vh-60">
+        <div class="bg-light border rounded p-4 my-5 w-50">
+          <h1 class="text-center">Exercise 02 - Palindrome</h1>
+
+          <div class="mb-3 row">
+            <label for="number" class="col-md-4 col-form-label"
+              >Enter a positive number:</label
+            >
+            <div class="col-md-8">
+              <input
+                class="form-control"
+                type="number"
+                id="number"
+                placeholder="Enter a positive number"
```

Comment:
I think this `placeholder` is redundant since it repeats the text in the `label` above. What do you think about removing it ❓

Reply:
Placeholder improves the accessibility and validation of HTML code that's why i kept!